### ISSUE TITLE
letters with accents changed to ? when inserting Data

### ISSUE THREAD

**Issue by elraider33** [schambers/fluentmigrator#653]
Hi:

I have Migator for insert data of countries and cities. When a City or Country name have an accent (i.e. Manáus, )  it puts ? instead á. Is there a configuration that i need to change or is there a bug in the Code?

Thanks for your answer

**Comment by tommarien** [comment]
@elraider33 Hi,

We will need a little bit more detail, if you want help on this

* Say us how you insert the data
* Gives us a clue about which database, which schema
* ...

**Comment by elraider33** [comment]
sure:
here's the code:

List<string> columns = new List<string>();
            using (var reader = new CsvFileReader(filename))
            {
                reader.ReadRow(columns); // skip column names
                while (reader.ReadRow(columns))
                {

                    Insert.IntoTable("Country").Row(
                        new { 
                            CountryId = columns[0], 
                            Country = columns[1], 
                            FIPS104 = columns[2], 
                            ISO2 = columns[3], 
                            ISO3  = columns[4],
                            ISON = columns[5],
                            Internet = columns[6],
                            Capital = columns[7],
                            MapReference = columns[8],
                            NationalitySingular = columns[9],
                            NationalityPlural = columns[10],
                            Currency = columns[11],
                            CurrencyCode = columns[12],
                            Population = columns[13],
                            Title = columns[14],
                            Comment = columns[15],
                            CountryCode = columns[16]
                        });
                }
            }

the source of the data is a CSV file, The Database is  SQL Azure, but also in my test enviroment (SQL Server 2008 with Latin1_General_CI_AS Collation) the result is the same. 
basically what i do is read each line of the CSV file and  insert it into the Database.

Let me know if you need anything else.

Thanks

**Comment by tommarien** [comment]
@elraider33 ok, can you check if your csvfilereader is not the culprit, could be some file encoding issue ?

**Comment by elraider33** [comment]
indeed it was the encoding. Thanks for the support. You can close this case.

**Comment by tommarien** [comment]
@elraider33 No problem, glad to assist ;)
